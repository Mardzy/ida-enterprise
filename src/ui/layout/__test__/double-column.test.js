import DoubleColumn from "../double-column";

describe("The double column component", () => {

  it("renders both sections", () => {
    const left = () => <div className="left">Left</div>;
    const right = () => <div className="right">Right</div>;

    const wrapper = mount(
      <DoubleColumn left={left} right={right}/>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });

});