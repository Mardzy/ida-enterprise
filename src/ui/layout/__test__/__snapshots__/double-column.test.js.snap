// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The double column component renders both sections 1`] = `
<DoubleColumn
  left={[Function]}
  right={[Function]}
>
  <Grid
    columns={2}
    stackable={true}
  >
    <div
      className="ui stackable two column grid"
    >
      <GridColumn
        computer={8}
        tablet={16}
      >
        <div
          className="eight wide computer sixteen wide tablet column"
        >
          <div
            className="left"
          >
            Left
          </div>
        </div>
      </GridColumn>
      <GridColumn
        computer={8}
        tablet={16}
      >
        <div
          className="eight wide computer sixteen wide tablet column"
        >
          <div
            className="right"
          >
            Right
          </div>
        </div>
      </GridColumn>
    </div>
  </Grid>
</DoubleColumn>
`;
